src/compas_timber/connections/ball_node.py

```diff
@@ -101,7 +105,10 @@ def _default_interface(self):
 
     @classmethod
     def create(cls, model, *elements, **kwargs):
```

Comment:
Shouldn't `elements` be passed directly instead of unpacked `(*elements)`, since `BallNodeJoint` expects a list of beams?

Reply:
I left the `create()` args exactly as they are in the generic `Joint.create()` method and changed the way it's handled inside the method.